spec/models/question_spec.rb

```diff
@@ -40,4 +40,22 @@
   describe 'attributes' do
     it { expect(question.answers).to include(*question.correct_answers) }
   end
+
+  describe 'public instance methods' do
+    context 'responds to its methods' do
+      it { expect(question).to respond_to(:correct_count) }
+      it { expect(question).to respond_to(:correct_ratio) }
+      it { expect(question).to respond_to(:total_rates) }
+      it { expect(question).to respond_to(:like_ratio) }
+    end
+
+    context 'executes methods correctly' do
+      context 'self.correct_ratio' do
```

Comment:
Two notes here:

- First: this method belongs to the presenter. IMHO, it would make sense to test it inside a presenter_spec file. What was the idea behind testing it here?
- Second: you can test the method with real values. Set the inputs, and check the outputs: this kind of maths method are pretty cool to unit test. :)